package.json

```diff
@@ -9,57 +9,50 @@
-    "txml": "^3.1.1"
+    "pako": "^1.0.11",
+    "regenerator-runtime": "^0.13.3",
+    "threads": "^1.3.1",
+    "txml": "^3.1.2"
   },
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
-    "babel-eslint": "^7.1.1",
-    "babel-loader": "^7.1.2",
-    "babel-plugin-syntax-async-functions": "^6.13.0",
-    "babel-plugin-transform-async-to-generator": "^6.16.0",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
-    "chai": "^3.5.0",
-    "detect-node": "^2.0.3",
-    "es6-promise": "^4.0.5",
-    "eslint": "^4.18.0",
-    "eslint-config-airbnb": "^16.1.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.7.0",
-    "express": "^4.16.3",
+    "@babel/core": "^7.8.7",
+    "@babel/register": "^7.8.6",
+    "chai": "^4.2.0",
+    "detect-node": "^2.0.4",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.19.0",
+    "express": "^4.17.1",
     "isomorphic-fetch": "^2.2.1",
-    "jsdoc": "^3.5.5",
-    "jsdoc-babel": "^0.2.1",
+    "jsdoc": "^3.6.3",
+    "jsdoc-babel": "^0.5.0",
     "jshint-stylish": "^2.2.1",
-    "mocha": "^3.1.2",
-    "webpack": "^4.41.6",
-    "webpack-bundle-analyzer": "^3.6.0",
-    "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.10.3",
-    "webpack-merge": "^4.2.2",
-    "worker-loader": "^2.0.0"
+    "mocha": "^7.1.0",
+    "parcel-bundler": "^1.12.4",
+    "parcel-plugin-bundle-visualiser": "^1.2.0"
   },
   "scripts": {
-    "analyze": "NODE_ENV=production webpack -p --config webpack.analyze.js",
-    "dev": "webpack-dev-server -d -w --content-base test",
-    "watch": "webpack -d -w",
-    "build:clean": "rm -rf dist/*",
-    "bundle:dev": "webpack -d ",
-    "bundle:prod": "NODE_ENV=production webpack -p",
-    "build:babel": "babel src/ -d dist",
-    "build:babel:watch": "babel src/ -d dist -w",
-    "build": "npm run build:clean ; npm run bundle:dev ; npm run bundle:prod ; npm run build:babel",
-    "test": "mocha --compilers js:babel-core/register --require babel-polyfill",
+    "build": "npm run build:clean; npm run build:browser; npm run build:node;",
+    "build:clean": "rm -rf dist-node/ dist-browser/",
+    "build:node": "parcel build src/main.js --target node --out-dir dist-node/",
+    "build:browser": "parcel build src/main.js --target browser --out-dir dist-browser/ --global GeoTIFF",
+    "dev": "parcel serve test/data/** test/index.html",
+    "dev:clean": "rm -rf dist/ .cache/",
+    "docs": "rm -rf docs/; jsdoc -c .jsdoc.json -r src README.md -d docs",
     "lint": "eslint src",
-    "docs": "rm -rf docs/*; jsdoc -c .jsdoc.json -r src README.md -d docs",
-    "prepublish": "npm run build",
-    "prepare": "npm run build"
+    "lint:fix": "eslint src --fix",
+    "prepublishOnly": "npm run build",
```

Comment:
The issue with `prepublish` hook is that it is executed for both `npm publish` and `npm install` and same for `prepare` but this runs `npm run build` twice if you run `npm install`. That's why it is recommended to use `prepublishOnly` to prevent the double build issue. 

While saying this, I realize that we should remove `prepublishOnly` and only use `prepare` as the latest also runs before `npm publish`. So is the current state of package.json `npm run build` runs twice if you use `npm publish`.
see: https://docs.npmjs.com/misc/scripts#deprecation-note for more details
and https://docs.npmjs.com/misc/scripts#description

Reply:
Ah, I see. So `prepublishOnly` is actually more fitting. 👍